README.md

```diff
@@ -136,3 +136,44 @@ In the `data` field is where you make your map. A space means empty space and a
 If you feel like there's something you'd like to add to the game, feel free to make a fork and submit a pull request! I'll try to review it as soon as possible. If there's an issue with how I've structured the code in the project and you feel like there's a better way, however, please make an issue instead. Hebi is a learning project for me to learn Rust and Bevy, and I'd like to implement most of the core changes myself.
 
 If you want to create a custom map generation type, a good starting point would be looking at `src/maps/default.rs`. Once you've got your generator working, submit it with a pull request! The more variety in game maps the better.
+
+
+## Building from source
+
+You've taken interest in the project and would like to work on it or build your own Hebi from source, follow through these steps to get it up and running on your machine.
```

Comment:
Feels a bit long-winded (literally!) Maybe split the sentence on the comma, and either:

 - Make the first part a question;
 - Add some preposition to the second part ("Then, follow through [...]", "Now, [...]" etc)

Reply:
> Make the first part a question

It would sound better if it was a question, yes.